packages/extension-ui/src/components/Spinner.tsx

```diff
@@ -17,15 +17,15 @@ function Spinner({ className = '', size = 'normal' }: Props): React.ReactElement
   return (
     <img
       className={`${className} ${size}Size`}
-      src={spinnerDotsSrc}
+      src={spinner}
     />
   );
 }
 
 export default React.memo(styled(Spinner)`
   bottom: 0rem;
-  height: 3rem;
   left: 50%;
+  top: 14%;
```

Comment:
Can't we be more specific than 14%? Seems like a very magic number in relations to other dimensions that can break in some scenarios 🤔